QUEST_JOBSTEP.tsv

```diff
@@ -305,21 +305,21 @@ QUEST_JOBSTEP_20150317_000304	Centurion Master
 QUEST_JOBSTEP_20150317_000305	The Centurion. It's not an easy path. {nl}But it's a different story if you win a duel with me. What do you think?
 QUEST_JOBSTEP_20150317_000306	The duel will take place in plaza vacant lot so be there first.
 QUEST_JOBSTEP_20150317_000307	That is enough. {nl}I will be responsible for teaching you.
-QUEST_JOBSTEP_20150317_000308	Sadhu's train by training your soul to become stronger. {nl}I will check though a duel...to see if your soul is prepared enough to take our trainings.
+QUEST_JOBSTEP_20150317_000308	Sadhu's train by training your soul to become stronger. {nl}I will check through a duel...to see if your soul is prepared enough to take on our training.
 QUEST_JOBSTEP_20150317_000309	Let's duel at the empty lot in the square.{nl}I will adjust to your level so don't worry too much.
-QUEST_JOBSTEP_20150317_000310	I can see your will. {nl}Welcome to joining the Sadhu.
+QUEST_JOBSTEP_20150317_000310	I can see your will. {nl}Welcome to the Sadhu.
 QUEST_JOBSTEP_20150317_000311	Hackapell Master
-QUEST_JOBSTEP_20150317_000312	It's good that you want to learn my combat skills but first, I need to check your abilities. {nl}I need to know your weakness in order to teach you, isn't it?
+QUEST_JOBSTEP_20150317_000312	It's good that you want to learn my combat skills, but first I need to check your abilities. {nl}I need to know your weakness in order to teach you, isn't it?
 QUEST_JOBSTEP_20150317_000313	I'll see you in the plaza vacant area. {nl}Be there first.
-QUEST_JOBSTEP_20150317_000314	This seems enough. {nl}Harsh trainings will begin from now.
+QUEST_JOBSTEP_20150317_000314	This seems enough. {nl}Harsh training will begin from now.
 QUEST_JOBSTEP_20150317_000315	Wugushi Master
 QUEST_JOBSTEP_20150317_000316	Wugushi s don't hesitate on using poison. {nl}Before you learn anything, I'll show you how we fight. {nl}Arrows or airguns, I'll show you with whatever it takes.
-QUEST_JOBSTEP_20150317_000317	Let's meet in Plaza Vacant Lot. {nl}This place is too small, others might get hurt.
+QUEST_JOBSTEP_20150317_000317	Let's meet in the plaza vacant area. {nl}This place is too small, others might get hurt.
```

Comment:
This line comes up a few times, I see it worded differently in a few instances, so I assume "plaza vacant area" is a point of interest rather than the actual name of a location.